heralding/capabilities/http.py

```diff
@@ -22,29 +22,23 @@
 
 import base64
 import logging
-from http.server import BaseHTTPRequestHandler
 
 from heralding.capabilities.handlerbase import HandlerBase
+from heralding.libs.http.aioserver import AsyncBaseHTTPRequestHandler
 
 logger = logging.getLogger(__name__)
 
 
-class BeeHTTPHandler(BaseHTTPRequestHandler):
-    def __init__(self, request, client_address, server, httpsession, options):
-
-        # Had to call parent initializer later, because the methods used
-        # in BaseHTTPRequestHandler.__init__() call handle_one_request()
-        # which calls the do_* methods here. If _banner, _session and _options
-        # are not set, we get a bunch of errors (Undefined reference blah blah)
-
+class BeeHTTPHandler(AsyncBaseHTTPRequestHandler):
```

Comment:
The Bee prefix is legacy stuff, can you please remove the prefix (or rename if it collides with builtin stuff)?